### ISSUE TITLE
Choose the visible modules of the calculator

### ISSUE THREAD

**Issue by Cyber-Thibaut** [UpsilonNumworks/Upsilon#75]
**Is your feature request related to a problem? Please describe.**
Some people do not use the RPN and Reader modules so they take up space in their calculator menu.

**Describe the solution you want**.
I suggest that from the Settings menu we can choose whether we want these modules to be displayed or not in case we need them.

**Comment by FaustinM** [comment]
Yes but it's just put the dust under the carpet (mettre la poussière sous le tapis), we need to put every app as external and choose what we want during the installation process

**Comment by Cyber-Thibaut** [comment]
This is not the same as doing this. It just means that people don't have to install the whole calculator again if they need the module they left behind during the first installation. But we can propose at the time of the installation, the version with the modules in external and the version with the choice via the parameters

**Comment by Overengined** [comment]
Why not let the user choose ? It wouldn't be that hard to ask befor installation "do you want to have RPN and the Reader app installed dormant on your calculator or not at all ?", would it ?

**Comment by Yaya-Cout** [comment]
I think we can try to add a submenu “Visible apps” submenu in the settings with a switch per app to choose the visible apps without reinstalling or rebuilding all the system…

**Comment by Overengined** [comment]
> I think we can try to add a submenu “Visible apps” submenu in the settings with a switch per app to choose the visible apps without reinstalling or rebuilding all the system…

That is indeed a possibilty, but it implies some rework on the way the main menu is displayed then, because it would be of the uttermost uglyness to just display a blank space where the icon would be. Intsead, the icon disposition would have to dynamically adapt. In fact, I see a potential here to even add the possibility to move the icons of the menu from inside the calculator. Rearanging the main menu layout would indeed be very nice, but that's another topic.
